index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Stylish&display=swap" rel="stylesheet">
+    <title>Trivia Game</title>
+</head>
+
+<body background="assets/images/trivia-background.jpg">
+    <div class="container">
+        <header class="head">TOTALLY TRIVIAL TRIVIA!</header>
+        <div id="wrapper">
+            <div class="timer">Time Remaining:</div>
+            <div id="display">00:00</div>
+
+            <div class="question"> Who won the first Super Bowl?</div>
+            <div id="buttons">
```

Comment:
The questions you want to ask should be in an array or obj for in the JS file to get the answers and score